sass/includes/_author-intro.scss

```diff
@@ -1,3 +1,26 @@
+@use "@nationalarchives/frontend/nationalarchives/tools/media";
+
+.tna-author {
+    align-items: center;
+
+    &__image {
+        margin: 0 auto;
+
+        overflow: hidden;
+        border-radius: 100%;
+
+        img {
+            width: 100%;
+            height: 100%;
+        }
+
+        @include media.on-tiny {
+            max-width: 15rem;
+            margin-bottom: 3rem;
+        }
```

Comment:
Another one for my own understanding: What does `media.on-tiny` do? Smallest screensize media query?

Reply:
In the `tna-frontend`, we have four classifications of device for media queries; `large`, `medium`, `small` and `tiny`. I picked these rather than `desktop`, `tablet` and `mobile` because I don't care about the device, only the available screen size. It is meant to abstract the *real* media queries away from you.

However, I needed names for "groups" of these so I could apply styles to more than one size at a time. `mobile` was what I decided to call the `small` and `tiny`-sized devices. Maybe a poor choice but I probably need to review this in the future.

The way media queries are named and grouped is described here is a crude ASCII diagram: https://github.com/nationalarchives/tna-frontend/blob/main/src/nationalarchives/tools/_media.scss

The actual values used are defined here: https://github.com/nationalarchives/tna-frontend/blob/main/src/nationalarchives/variables/_media.scss

At a point, the screen is too narrow to squeeze the image next to the title so I stack them. The CSS above just gives some extra space when that situation occurs and sets a maximum image size so it doesn't fill the whole mobile screen.